experiments/openebs/openebs-pool-disk-loss/openebs_pool_disk_loss_k8s_job.yml

```diff
@@ -0,0 +1,105 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  generateName: disk-loss-
+spec:
+  template:
+    metadata:
+      labels:
+        name: disk-loss
+    spec:
+      serviceAccountName: %CHAOS_SERVICE_ACCOUNT%
+      restartPolicy: Never
+      containers:
+      - name: ansibletest
+        image: litmuschaos/ansible-runner:ci
+        imagePullPolicy: Always
+        env:
+          - name: ANSIBLE_STDOUT_CALLBACK
+            value: 'default'
+
+          # Enter the App_label, if you set APP_CHECK to true
+          - name: APP_LABEL
+            value: 'app=mysql'
+
+          # Enter the APP_NAMESPACE, if you set APP_CHECK to true
+          - name: APP_NAMESPACE
+            value: 'default'
+            
+          - name: APP_PVC
+            value: ''
+
+          # This is a chaos namespace which will create all infra chaos resources in that namespace
+          - name: CHAOS_NAMESPACE
+            valueFrom:
+              fieldRef:
+                fieldPath: metadata.namespace
+
+          # Only GCP and AWS is supported
+          - name: CLOUD_PLATFORM
+            value: 'GCP'
+
+          # Enter the time duration after the disk is dettached
+          - name: TOTAL_CHAOS_DURATION
+            value: '10'
```

Comment:
Can you please put a comment stating about it's unit?

Reply:
Done